pkg/inference/backends/vllm/vllm.go

```diff
@@ -3,16 +3,30 @@ package vllm
 import (
 	"context"
 	"errors"
+	"fmt"
+	"io"
+	"io/fs"
 	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
 
+	"github.com/docker/model-runner/pkg/diskusage"
 	"github.com/docker/model-runner/pkg/inference"
 	"github.com/docker/model-runner/pkg/inference/models"
+	"github.com/docker/model-runner/pkg/inference/platform"
+	"github.com/docker/model-runner/pkg/internal/utils"
 	"github.com/docker/model-runner/pkg/logging"
+	"github.com/docker/model-runner/pkg/sandbox"
+	"github.com/docker/model-runner/pkg/tailbuffer"
 )
 
 const (
 	// Name is the backend name.
-	Name = "vllm"
+	Name    = "vllm"
+	vllmDir = "/opt/vllm-env/bin"
```

Comment:
I am sometimes curious why we care about the directory in the golang code. We could just add this directory to the PATH in the Dockerfile.

Could be useful if we start to use artifacts from this container for example for ROCm which expect things to be in a different path:

https://hub.docker.com/r/rocm/vllm

can worry about this in a follow on PR though

Reply:
I've done it like this to follow llama.cpp's approach where it can be installed on the host machine, where we can't have anything we want in the PATH.